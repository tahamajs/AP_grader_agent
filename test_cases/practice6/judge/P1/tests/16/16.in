POST signup ? username "babak" password "kabab"
POST login ? username "babak" password "kabab"
GET restaurants ?
GET restaurants ? food_name "Kabab Barg"
PUT my_district ? district "Dowlat Abad"
GET restaurants ?
GET restaurants ? food_name "pitza"