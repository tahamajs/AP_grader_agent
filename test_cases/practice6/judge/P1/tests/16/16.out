OK
Permission Denied
Not Found
Not Found
OK
eram (Dowlat Abad)
majid (Payane Jonoub)
gardino (Pirouzi)
niko sefat (Enghelab)
shila2 (Enghelab)
shiraz (Enghelab)
yas (Enghelab)
burger land (Tehranpars)
shila3 (Tehranpars)
hossein garden (Hossein Abad)
martin (Karegar)
orkideh (Karegar)
hani (Mirdamad)
tehroonchi (Mirdamad)
khorsand (Tajrish)
rouhi (Tajrish)
shila1 (Tajrish)
fish & chips (Meydan Sanat)
lidoma (Meydan Sanat)
sib (Meydan Sanat)
majid (Payane Jonoub)
gardino (Pirouzi)
shila2 (Enghelab)
shiraz (Enghelab)
burger land (Tehranpars)
shila3 (Tehranpars)
martin (Karegar)
shila1 (Tajrish)
sib (Meydan Sanat)
